code/game/objects/structures/industrial_lift.dm

```diff
@@ -485,7 +485,7 @@ GLOBAL_LIST_EMPTY(lifts)
 	var/obj/effect/landmark/tram/from_where
 	var/travel_direction
 
-GLOBAL_DATUM(central_tram, /obj/structure/industrial_lift/tram/central)
+GLOBAL_LIST_EMPTY_TYPED(central_trams, /obj/structure/industrial_lift/tram/central)
```

Comment:
This global list of central trams is decided by z-level, but industrial lifts have the capability of going up, down, or even diagonally through z-levels. For that reason, we can't have this decide via z-level. Choose an alternate solution, pls!